ckanext/validation/settings.py

```diff
@@ -1,15 +1,8 @@
 # encoding: utf-8
 
-from ckantoolkit import config
-
-# TODO: configurable
+SUPPORTED_FORMATS = u"ckanext.validation.formats"
 DEFAULT_SUPPORTED_FORMATS = [u'csv', u'xls', u'xlsx']
-
-SUPPORTED_FORMATS = config.get(
-    u'ckanext.validation.formats', DEFAULT_SUPPORTED_FORMATS)
-DEFAULT_VALIDATION_OPTIONS = config.get(
-    u'ckanext.validation.default_validation_options', '{}'
-)
+DEFAULT_VALIDATION_OPTIONS = "ckanext.validation.default_validation_options"
```

Comment:
It would be better to call this something like `DEFAULT_VALIDATION_OPTIONS_KEY`.

Reply:
\+